com.unity.render-pipelines.high-definition/Runtime/Compositor/CompositionManager.cs

```diff
@@ -496,6 +495,8 @@ public void UpdateLayerSetup()
             SetupLayerPriorities();
         }
 
+        static HDRenderPipelineGlobalSettings m_globalSettings;
```

Comment:
If I understand well, there is case where we want to change the global settings (using Unity in dofferent context). If it is registered as static here, it will not adapt except if a domain reload occures. Can we assert this is the case?